app/utilities/schema.py

```diff
@@ -25,6 +26,37 @@
 }
 
 
+def get_schema_path_map_for_language(language_code):
+    schema_files = []
+
+    for schema_dir in SCHEMA_DIRS:
+        schema_files.extend(glob(f"{schema_dir}/{language_code}/*.json"))
+
+    return {
+        Path(schema_file).with_suffix("").name: schema_file
+        for schema_file in schema_files
+    }
+
+
+def get_schema_path_map():
```

Comment:
```suggestion
def get_schema_path_maps():
```

Reply:
There's only a single map returned from this method.